default/it/cat_printed.md

```diff
@@ -0,0 +1,151 @@
+## **Copertura delle edizioni musicali a stampa in RISM**
+
+RISM ambisce a documentare edizioni musicali a stampa dai primordi della stampa musicale fino al 1900 circa.
+
+Nel corso del XX secolo e fino ad oggi si è assistito a un forte incremento della pubblicazione e distribuzione commerciale della musica. La quantità di musica considerata "rara" o non documentata altrove è molto minore rispetto ai secoli precedenti. La musica pubblicata da editori moderni, stampata in ampie tirature, facilmente disponibile, e destinata dai gestori di biblioteche a un uso pratico è documentata più opportunamente in banche-dati collettive come [WorldCat](http://www.worldcat.org/).
+
+RISM può considerare caso per caso l'opportunità di documentare musica pubblicata nel XX secolo da editori scomparsi o editori minori e regionali d'importanza storica; o anche edizioni moderne incluse nel contesto della catalogazione di un'intera collezione di valore storico, ad esempio la biblioteca privata di un individuo.
+
+## **Quando inserire una nuova scheda (edizioni musicali a stampa)**
+
+RISM tratta ogni manoscritto come un oggetto unico e perciò ogni manoscritto la la sua scheda RISM. Il materiale a stampa è però differente, in quanto si possono descrivere copie multiple della stessa edizione utilizzando la stessa scheda (aggiungendo semplicemente i tuoi dati di possesso), ma varianti significative richiedono la creazione di una nuova scheda.
+
+Qui di seguito si offre una guida per decidere quali varianti giustificano una nuova scheda e quali no. (Questa sezione riprende ampiamente da [Descriptive Cataloging of Rare Materials (Music)](http://rbms.info/dcrm/)).
+
+**È necessaria una nuova scheda**  se il tuo oggetto mostra una o più delle differenze seguenti, in confronto alla scheda RISM esistente:
+
+-
+
+**Contenuto diverso**: Differenze nel titolo o negli autori/contributori. Dati editoriali che indicano correzioni, revisioni, ampliamenti, abbreviazioni o l'inclusione di materiale supplementare.
+
+- **Editore diverso**.
+-
+
+**Diversa presentazione tipografica**: Cambiamenti nell'estensione. Modifica del formato bibliografico o musicale. Differenze nel cambio di righe e nel richiamo (nel paragone tra copie diverse).
+
+-
+
+**Statuto editoriale diverso**: Un frontespizio originale è stato cancellato e rimpiazzato. Una nuova etichetta copre i dati di pubblicazione, distribuzione, produzione ecc. Le pagine originali sono state pubblicate in una nuova coperta che reca informazioni più recenti di quelle riportate sul frontespizio. È stato aggiunto un frontespizio di una serie.
+
+-
+
+**Numero di lastra e/o numero editoriale diverso**.
+
+**Non creare una nuova scheda**  se il tuo oggetto mostra una delle differenze seguenti. (Se il tuo oggetto mostra più di una di queste differenze, devi decidere se è necessaria una nuova scheda.)
+
+-
```

Comment:
I think the formatting of this list is messed up

Reply:
Probably. Will do that in a separate phase.